README.md

```diff
@@ -1,16 +1,31 @@
-# Edit The Docs
+# Contribute to Rancher Docs
 
-To get started, fork and clone the rancher-docs repository.
+Welcome to the [Rancher docs](https://ranchermanager.docs.rancher.com/) repository. See the [Rancher software](https://github.com/rancher/rancher) repository if you have questions or requests for the Rancher platform.
+
+## Make a Suggestion
+
+You can suggest changes to the Rancher docs in two ways: 
+
+1. [Open an issue](https://github.com/rancher/rancher-docs/issues/new/choose).
+1. Edit the docs in the way you see fit and open a pull request.
```

Comment:
I don't think it should be mentioned here, because then we risk getting caught describing how to open a PR in general. I assumed the person reading this file would already be familiar with Git, and there are at least 3 ways to edit a file (Github, docs site -> Github, or from your desktop). 

Maybe it would make sense to add that to https://ranchermanager.docs.rancher.com/v2.7/contribute-to-rancher though, since the reader would already be on the docs site to click that button.